City-of-Helsinki/drupal-module-helfi-proxy
# UHF-9493: Drupal 10.3 compatibility

## Drupal 10.3 update

❗ Note that the tests will fail until the Helfi Drupal Platform PR https://github.com/City-of-Helsinki/drupal-helfi-platform/pull/252 is merged ❗ 

## What was done
- Removed Language negotiation patch as it was fixed for [Drupal 10.3](https://www.drupal.org/project/drupal/issues/3385550).

## How to test
- Check testing instructions from https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/780

## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-platform/pull/252
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/780
* https://github.com/City-of-Helsinki/drupal-module-helfi-proxy/pull/73
* https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/653
